tests/test_registries.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import markers
+
+from pages.registries import (
+    RegistriesLandingPage,
+    RegistriesDiscoverPage,
+    RegistrationDetailPage
+)
+
+@pytest.fixture
+def landing_page(driver):
+    landing_page = RegistriesLandingPage(driver)
+    landing_page.goto()
+    return landing_page
+
+class TestRegistriesLandingPage:
+    @markers.smoke_test
+    @markers.core_functionality
+    def test_landing_page_loads(self, driver):
+        RegistriesLandingPage(driver).goto()
+
+class TestRegistriesDiscoverPage:
+
+    @markers.core_functionality
+    def test_discover_page_loads(self, driver):
+        RegistriesDiscoverPage(driver).goto()
+
+    @markers.smoke_test
+    @markers.core_functionality
+    def test_search_results_exist(self, driver, landing_page):
+        landing_page.search_button.click()
+        discover_page = RegistriesDiscoverPage(driver, verify=True)
+        discover_page.loading_indicator.here_then_gone()
+        assert len(discover_page.search_results) > 0
+
+    def test_detail_page(self, driver):
```

Comment:
This test looks good, but I'm not sure it actually tests what we said it tests based on the locators above.